notifications_utils/field.py

```diff
@@ -16,17 +18,45 @@ def __init__(self, body):
         # body shouldn’t include leading/trailing brackets, like (( and ))
         self.body = body.lstrip("(").rstrip(")")
 
+    class Types(StrEnum):
+        BASE = auto()
+        UNSAFE = auto()
+        CONDITIONAL = auto()
+
+    extended_type_pattern = OrderedDict(
```

Comment:
`dict`s are always ordered [as of Python 3.7](https://mail.python.org/pipermail/python-dev/2017-December/151283.html) so `OrderedDict` [is not needed](https://github.com/alphagov/notifications-utils/pull/958)

Reply:
removed - just a standard dict now